src/Verifier/SAW/Grammar.y

```diff
@@ -246,17 +247,17 @@ data ParserState = PS { psInput :: AlexInput
                       , psErrors :: [PosPair ParseError]
```

Comment:
I don't think we need this `ParserState` datatype anymore, or the `ErrorList` type.

Reply:
I removed those, and some other dead code, in a07d0a8.